concent_api/concent_api/settings/base.py

```diff
@@ -17,6 +17,9 @@
     'concent_api',
     'core',
     'gatekeeper',
+
+    # Add raven plugin to support sentry.io
+    'raven.contrib.django.raven_compat',
```

Comment:
Put this above our apps. Let's keep this order here: django apps, external apps, our apps.

Reply:
Done.